src/routes/reports/targets/Targets.tsx

```
@@ -0,0 +1,106 @@
+    },
+    sorters: {
+      initial: [
+        { field: "month", order: "desc" },
+        { field: "year", order: "desc" },
+      ],
+    },
+  });
+
+  return (
+    <Show headerButtons={[
+      <Button onClick={() => go({ to: `/administration/reports/targets/create`,query: { user_id: pathname.split("/").pop() } })}>Create Target</Button>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)